src/components/DragAndDrop/index.js

```diff
@@ -0,0 +1,155 @@
+                ? window.innerWidth : boundingClientRect.right,
+            top: boundingClientRect.top,
+            bottom: boundingClientRect.bottom,
+        };
+    }
+
+    /**
+     * @param {Object} event native Event
+     */
+    dropZoneDragHandler(event) {
+        // Setting dropEffect for dragover is required for '+' icon on certain platforms/browsers (eg. Safari)
+        switch (event.type) {
+            case 'dragover':
+                // Continuous event -> can hurt performance, be careful when subscribing
+                // eslint-disable-next-line no-param-reassign
+                event.dataTransfer.dropEffect = COPY_DROP_EFFECT;
+                this.dragOverHandler(event);
+                break;
+            case 'dragenter':
+                // Avoid reporting onDragEnter for children views -> not performant
+                if (this.dropZoneDragState === 'dragleave') {
+                    this.dropZoneDragState = 'dragenter';
+                    // eslint-disable-next-line no-param-reassign
+                    event.dataTransfer.dropEffect = COPY_DROP_EFFECT;
+                    this.props.onDragEnter(event);
+                }
+                break;
+            case 'dragleave':
+                if (this.dropZoneDragState === 'dragenter') {
+                    if (
+                        event.clientY <= this.dropZoneRect.top
+                                || event.clientY >= this.dropZoneRect.bottom
+                                || event.clientX <= this.dropZoneRect.left
+                                || event.clientX >= this.dropZoneRect.right
+
+                                // Cancel drag when file manager is on top of the drop zone area - works only on chromium
+                                || (event.target.getAttribute('id') === this.props.activeDropZoneId && !event.relatedTarget)
+                    ) {
+                        this.dropZoneDragState = 'dragleave';
+                        this.props.onDragLeave(event);
+                    }
+                }
+                break;
+            case 'drop':
+                this.dropZoneDragState = 'dragleave';
+                this.props.onDrop(event);
+                break;
+            default: break;
+        }
+    }
+
+    /**
+     * Handles all types of drag-N-drop events on the drop zone associated with composer
+     *
+     * @param {Object} event native Event
+     */
+    dropZoneDragListener(event) {
+        event.preventDefault();
+        if (this.dropZone.contains(event.target)) {
+            this.dropZoneDragHandler(event);
+        } else {
+            // eslint-disable-next-line no-param-reassign
+            event.dataTransfer.dropEffect = NONE_DROP_EFFECT;
+        }
+    }
+
+    render() {
+        return this.props.children;
+    }
+}
+
+DragAndDrop.propTypes = propTypes;
+DragAndDrop.defaultProps = defaultProps;
+
+export {
+    DragAndDropPropTypes,
+};
```

Comment:
This should be removed as well. Now all the imports of `DragAndDropPropTypes` will refer to the new file you created.